dashboard/app/models/script.rb

```diff
@@ -1409,7 +1409,7 @@ def summarize(include_lessons = true, user = nil, include_bonus_levels = false)
       isHocScript: hoc?,
       csf: csf?,
       peerReviewsRequired: peer_reviews_to_complete || 0,
-      peerReviewStage: peer_review_stage,
+      peerReviewLessonInfo: peer_review_lesson_info,
```

Comment:
On one of my PRs @davidsbailey mentioned that the old javascript might stay cached in browsers, and that we should keep the old value in the `summarize` data for a few days until it's likely to be cleared. Dave, do you think that applies here also?

If we're not sure, wouldn't hurt to keep it for now to be safe.

Reply:
+1, this is a good practice to follow. there are a number of ways to slice it. the important part is that in each PR/deploy, the old JS should work with the new server code.

Reply:
...that said if we're sure this is something that only gets sent down on page load (as opposed to an ajax request) then it could be ok to skip this, since the client would get latest JS when loading or reloading the page.

Reply:
Do that mean I should have both like:
```
peerReviewStage: peer_review_lesson_info,
peerReviewLessonInfo: peer_review_lesson_info,
```
??

Reply:
correct! as long as the data is not too big, the simplest way is to duplicate it in the server response.